server.js

```diff
@@ -64,13 +64,123 @@ github.authenticate({
 var app = express();
 
 var dlangReviewers = {
-    "dlang/dmd": ["WalterBright", "MartinNowak", "yebblies", "AndrejMitrovic", "klickverbot", "9rnsr",
-            "andralex"],
-    "dlang/druntime": ["WalterBright", "MartinNowak", "yebblies", "AndrejMitrovic", "klickverbot", "9rnsr",
-            "andralex", "DmitryOlshansky", "jmdavis", "rainers", "schveiguy", "CyberShadow"],
-    "dlang/phobos": ["WalterBright", "MartinNowak", "yebblies", "AndrejMitrovic", "klickverbot", "9rnsr",
-            "andralex", "DmitryOlshansky", "jmdavis", "rainers", "schveiguy", "9il", "CyberShadow",
-                "Hackerpilot", "quickfur", "wilzbach"],
+    "dlang/dmd": [
+        "9rnsr",
+        "andralex",
+        "AndrejMitrovic",
+        //"AndrewEdwards",
+        "braddr",
```

Comment:
He is still on the blacklist (see below)  as we haven't implented a time-based scoring yet.